UI/GameSettingsScreen.cpp

```diff
@@ -319,7 +319,7 @@ void GameSettingsScreen::CreateViews() {
 	PopupMultiChoice *beziersChoice = graphicsSettings->Add(new PopupMultiChoice(&g_Config.iSplineBezierQuality, gr->T("LowCurves", "Spline/Bezier curves quality"), quality, 0, ARRAY_SIZE(quality), gr->GetName(), screenManager()));
 	beziersChoice->OnChoice.Add([=](EventParams &e) {
 		if (g_Config.iSplineBezierQuality != 0) {
-			settingInfo_->Show(gr->T("LowCurves Tip", "Improve/Reduce the quality of rendered splines and bezier curves"), e.v);
+			settingInfo_->Show(gr->T("LowCurves Tip", "Only used in some effects, and circles like LocoRoco"), e.v);
```

Comment:
Hmm, right.  Maybe "Only used by some games, increases smoothness of curves"?

-[Unknown]